backend/src/main/java/com/mojprofesor/backend/entity/OpinionEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.mojprofesor.backend.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "opinion")
+public class OpinionEntity {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private long professorID;
```

Comment:
Powinno być

@ManyToOne
private Professor professor;

https://www.baeldung.com/spring-data-rest-relationships#1-the-data-model-1

Reply:
Encja z profesorem jest stworzona jak coś.